### DISCUSSION TITLE
Corporate Shared folder analysis

### DISCUSSION THREAD

**Discussion by tardispilot** [FuLoRi/Analyze-LastPassVaultGUI#14] [Q&A]
We are a long-time corporate user of LastPass, leveraging the shared folder feature. The analysis tool is great for "My Vault" but the initial XML dump does not contain any of the shared content. Is there a way to extend the utility to examine any sites saved within shared folders too?

**Comment by FuLoRi** [comment]
I ran across the same thing. It's not an issue with the script, but rather with the method used to dump the XML from LastPass. I don't know how to work around it yet, but maybe someone out there will let us know. 🤞
